### ISSUE TITLE
Poor performance on PointOdyssey

### ISSUE THREAD

**Issue by snowyyyyy123** [CUT3R/CUT3R#70]
Thanks for the excellent paper and code. I noticed that the results on the PointOdyssey test set, even after finetuning on the entire PointOdyssey train set, seem to be pretty poor. Did you notice this, and do you have any idea why this is happening?

**Comment by Forrest-110** [comment]
Hi @beyondthefryingpan! We haven’t focused much on the PointOdyssey test set since it's a synthetic dataset and includes certain extreme complex scenarios (e.g., foggy environments).

**Comment by haonanhe** [comment]
@beyondthefryingpan Do you have any idea why this is happening? I also find the same problem. Does this cause by the depth scale of PointOdyssey different from other datasets?

**Comment by zhangshuoneu** [comment]
> Thanks for the excellent paper and code. I noticed that the results on the PointOdyssey test set, even after finetuning on the entire PointOdyssey train set, seem to be pretty poor. Did you notice this, and do you have any idea why this is happening?

Maybe there are some error in PointOdyssey according to https://github.com/CUT3R/CUT3R/issues/55

**Comment by snowyyyyy123** [comment]
@haonanhe I am not sure. It may be because PointOdyssey has significant camera and object motion and CUT3R may not suited for this. @zhangshuoneu There may be errors but I don't think that's entirely the reason for the poor performance because other works have done well on this dataset.
